tests/apis/prv_api/test_service_areas.py

```diff
@@ -151,7 +152,7 @@ def test_all_polygons_get(client):
 def test_polygon_creation(client):
     response = client.post(
         POLY_BASE_URL,
-        data={"label": "test", "geom": MOCK_GEOJSON, "areas": []},
+        data={"label": "test", "geom": json.dumps(MOCK_GEOJSON), "areas": []},
```

Comment:
`str()` ? :sadpanda:

Reply:
`geom` field is expecting a JSON formated string with double quote. `str` use simple quote and raise exception.